docs(rename): rename Armory Spinnaker

317brian opened the pull request:
👋 

Can you give this a sanity check @stuposluns

317brian committed:
rename

317brian committed:
Update overview.md

stuposluns commented:
Spot checked, though I did not review every single change (I can, if that is helpful). Overall looks good to me. Would want @dnilasor to glance as well. Don't know if Armory Extended Halyard really works, but I'm not sure we have an issue with using "Halyard" in the first place.

aimeeu commented on content/en/docs/installation/armory-halyard.md:
```diff
@@ -5,11 +5,11 @@ description: >
 ---
 
-Armory Halyard is a versatile command line interface (CLI) to configure and deploy Spinnaker. It is made of a CLI that connects to a daemon.
+Armory extended Halyard is a versatile command line interface (CLI) to configure and deploy Spinnaker. It is made of a CLI that connects to a daemon.
```
Armory-extended Halyard

aimeeu commented on content/en/docs/installation/guide/aws-container-marketplace.md:
```diff
@@ -13,11 +13,11 @@ Please contact [Armory](mailto:hello@armory.io) if you're interested in an AWS M
 ## Overview
 
-Spinnaker Operator is a Kubernetes Operator that makes it easier to install, deploy, and upgrade Armory Spinnaker. The AWS Container Marketplace offering for Spinnaker Operator installs the Armory Spinnaker Operator in an EKS cluster. After that, Spinnaker can be installed in any namespace in your EKS cluster; this document assumes that Spinnaker will be installed in the `spinnaker` namespace.
+The Spinnaker Operator is a Kubernetes Operator for Spinnaker that makes it easier to install, deploy, and upgrade Spinnaker. The AWS Container Marketplace offering for Spinnaker Operator installs a version of the Spinnaker Operator extended by Armory in an EKS cluster. After that, Spinnaker can be installed in any namespace in your EKS cluster; this document assumes that Spinnaker will be instal ...
```
Re the Operator - Spinnaker Operator installs Spinnaker and Armory Operator installs Armory. Should we clarify to something like:

_The Spinnaker Operator is a Kubernetes Operator for Spinnaker that makes it easier to install, deploy, and upgrade open source Spinnaker. The Armory Operator extends the Spinnaker Operator to install Armory. The AWS Container Marketplace offering for Armory Operator installs the Operator in an EKS cluster. After that, Armory can be installed in any namespace in your EKS cluster; this document assumes that Armory will be installed in the `spinnaker` namespace._

aimeeu replied:
I looked up our offering in the AWS Container Marketplace. 
https://s.armory.io/2NuymGXd

The Armory Operator installs Armory, so we probably should omit all references to the open source Spinnaker Operator and open source Spinnaker. I can pull down this PR and update this aws-container-marketplace page if you like.

317brian replied:
Sure, this was a mass find/replace job, so there's bound to be edge cases like this page.

aimeeu commented on content/en/docs/installation/guide/aws-container-marketplace.md:
```diff
@@ -30,8 +30,8 @@ To use the Armory Spinnaker Marketplace offering, make sure you meet the followi
 1. Creating and configuring the necessary AWS IAM roles for your Kubernetes cluster
-2. Installing the Armory Spinnaker Custom Resource Definitions (CRDs) into your Kubernetes cluster
-3. Installing the Armory Spinnaker Operator
+2. Installing Custom Resource Definitions (CRDs) for Spinnaker into your Kubernetes cluster
```
should we replace Spinnaker with Armory?

aimeeu commented on content/en/docs/installation/guide/aws-container-marketplace.md:
```diff
@@ -30,8 +30,8 @@ To use the Armory Spinnaker Marketplace offering, make sure you meet the followi
-2. Installing the Armory Spinnaker Custom Resource Definitions (CRDs) into your Kubernetes cluster
-3. Installing the Armory Spinnaker Operator
+2. Installing Custom Resource Definitions (CRDs) for Spinnaker into your Kubernetes cluster
+3. Installing the Operator
```
Installing the Armory Operator ??

aimeeu commented on content/en/docs/installation/guide/aws-container-marketplace.md:
````diff
@@ -290,7 +290,7 @@ For example:
 ```
 
-## Installing Armory Spinnaker Custom Resource Definitions (CRDs)
+## Installing Spinnaker Custom Resource Definitions (CRDs)
````
Armory instead of Spinnaker?

317brian replied:
uhh thinking through this some more, it's technically Operator CRDs for Spinnaker right?

aimeeu commented on content/en/docs/installation/guide/aws-container-marketplace.md:
```diff
@@ -302,9 +302,9 @@ bash -c 'curl -L https://github.com/armory/marketplace/releases/latest/download/
 
-## Installing the Armory Spinnaker Operator
+## Installing the Spinnaker Operator
 
```
Armory Operator?

aimeeu commented on content/en/docs/installation/guide/configure-halyard.md:
```diff
@@ -7,7 +7,7 @@ aliases:
 ## Overview
 
-Armory Halyard can be configured via `/opt/spinnaker/config/halyard.yml`. If you run the Docker image, you can provide your own configuration by mounting the file or directory to the container. If you're running the Operator, you can also configure the behavior of the internal Halyard by creating a Kubernetes ConfigMap and mounting it to the Halyard container.
+Armory extended Halyard can be configured via `/opt/spinnaker/config/halyard.yml`. If you run the Docker image, you can provide your own configuration by mounting the file or directory to the container. If you're running the Operator, you can also configure the behavior of the internal Halyard by creating a Kubernetes ConfigMap and mounting it to the Halyard container.
```
Armory-extended Halyard

aimeeu commented on content/en/docs/installation/guide/configure-halyard.md:
```diff
@@ -30,7 +30,7 @@ You can choose a different location for your Spinnaker configuration by changing
 
 ## Versions
-Armory Halyard stores all the versions in a public s3 bucket (`halconfig`). Sometimes, clients prefer to store the versions in a storage under their control.
+Armory extended Halyard stores all the versions in a public s3 bucket (`halconfig`). Sometimes, clients prefer to store the versions in a storage under their control.
```
Armory-extended Halyard

aimeeu commented on content/en/docs/installation/guide/configure-halyard.md:
```diff
@@ -43,7 +43,7 @@ spinnaker:
 
 ### Using a private s3 bucket
-By default Armory Halyard will access version definitions and bills of materials without using the host's s3 credentials. You can force it to sign the s3 requests by adding:
+By default Armory extended Halyard will access version definitions and bills of materials without using the host's s3 credentials. You can force it to sign the s3 requests by adding:
```
Armory-extended Halyard

aimeeu commented on content/en/docs/installation/guide/configure-halyard.md:
````diff
@@ -59,11 +59,11 @@ docker run --name armory-halyard --rm \
 ```
 
-Replace `<Armory Halyard version>` with the version of Halyard you want to run.
+Replace `<Armory extended Halyard version>` with the version of Halyard you want to run.
````
Armory-extended Halyard

aimeeu commented on content/en/docs/installation/guide/configure-halyard.md:
```diff
@@ -59,11 +59,11 @@ docker run --name armory-halyard --rm \
 ### Using a private s3 bucket with assume role
 
-Armory Halyard can be configured to assume a specified role when accessing the bucket:
+Armory extended Halyard can be configured to assume a specified role when accessing the bucket:
```
Armory-extended Halyard

aimeeu commented on content/en/docs/installation/guide/install-on-aks.md:
```diff
@@ -236,7 +236,7 @@ cp kubeconfig-spinnaker-system-sa ${WORKING_DIRECTORY}/.secret
 ## Start the Halyard container
 
-On the `Halyard machine`, start the Halyard container (see the `armory/halyard-armory` [tag list](https://hub.docker.com/r/armory/halyard-armory/tags)) for the latest Armory Halyard Docker image tag.
+On the `Halyard machine`, start the Halyard container (see the `armory/halyard-armory` [tag list](https://hub.docker.com/r/armory/halyard-armory/tags)) for the latest Armory extended Halyard Docker image tag.
```
Armory-extended Halyard

aimeeu commented on content/en/docs/installation/guide/install-on-aws.md:
```diff
@@ -314,7 +314,7 @@ cp kubeconfig-spinnaker-system-sa ${WORKING_DIRECTORY}/.secret
 ## Start the Halyard container
 
-On the `docker machine`, start the Halyard container (see the `armory/halyard-armory` [tag list](https://hub.docker.com/r/armory/halyard-armory/tags)) for the latest Armory Halyard Docker image tag.
+On the `docker machine`, start the Halyard container (see the `armory/halyard-armory` [tag list](https://hub.docker.com/r/armory/halyard-armory/tags)) for the latest Armory extended Halyard Docker image tag.
```
Armory-extended Halyard

aimeeu commented on content/en/docs/installation/guide/install-on-gke-operator.md:
```diff
@@ -8,7 +8,7 @@ aliases:
 _Note: This guide is a work in progress._
 
-This guide contains instructions for installing Armory Spinnaker on a GKE Cluster using the [Spinnaker Operator]({{< ref "operator" >}}). Refer to the [Spinnaker Operator Reference]({{< ref "operator-config" >}}) for manifest entry details.
+This guide contains instructions for installing Armory on a GKE Cluster using the [Spinnaker Operator]({{< ref "operator" >}}). Refer to the [Spinnaker Operator Reference]({{< ref "operator-config" >}}) for manifest entry details.
```
Armory Operator ??

aimeeu commented on content/en/docs/installation/guide/install-on-gke.md:
```diff
@@ -227,7 +227,7 @@ cp spinnaker-gcs-account.json ${WORKING_DIRECTORY}/.secret
 ## Start the Halyard container
 
-On the `docker machine`, start the Halyard container (see the `armory/halyard-armory` [tag list](https://hub.docker.com/r/armory/halyard-armory/tags)) for the latest Armory Halyard Docker image tag.
+On the `docker machine`, start the Halyard container (see the `armory/halyard-armory` [tag list](https://hub.docker.com/r/armory/halyard-armory/tags)) for the latest Armory extended Halyard Docker image tag.
```
Armory-extended Halyard

aimeeu commented on content/en/docs/installation/guide/install-on-k8s.md:
````diff
@@ -726,7 +726,7 @@ You can get a list of available versions of Spinnaker with this command:
 ```
 
-* If you are installing Armory Spinnaker using Armory's Halyard, the command returns a version that starts with `2.x.x`
+* If you are installing Armory using Armory's Halyard, the command returns a version that starts with `2.x.x`
````
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/_index.md:
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Armory Halyard Release Notes"
+title: "Armory extended Halyard Release Notes"
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/_index.md:
```diff
@@ -10,15 +10,15 @@ aliases:
-## Armory Halyard
-Armory Halyard is an extended version of Halyard that deploys Armory features. You can find the base OSS version:
+## Halyard
+Armory extended Halyard  is an extended version of Halyard that deploys Armory features. You can find the base OSS version:
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/_index.md:
````diff
@@ -10,15 +10,15 @@ aliases:
 ```
 
-## List of Halyard Armory Releases
+## List of extended Halyard Releases
````
should this be Armory-extended Halyard Releases?

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-6-0.md:
```diff
@@ -14,7 +14,7 @@ aliases:
 - `hal shutdown` returns an `Error 400` message. We will be submitting a PR to OSS Spinnaker to resolve this.
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-6-1.md:
```diff
@@ -19,7 +19,7 @@ This release updates `kubectl` to `1.12.9` and fixes the version displayed from
 - `hal shutdown` returns an `Error 400` message. A PR has been submitted to OSS Spinnaker to resolve this. 
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-6-2.md:
```diff
@@ -19,7 +19,7 @@ This release fixes a regression introduced in `1.6.1` where `aws-iam-authenticat
 - None
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-6-3.md:
```diff
@@ -21,7 +21,7 @@ This release:
 - None
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-6-4.md:
```diff
@@ -22,9 +22,9 @@ This release:
 Halyard fails to generate the Secrets config block for some service profiles.
 
-Please upgrade to `Armory Halyard 1.6.5`.
+Please upgrade to `Armory extended Halyard 1.6.5`.
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-6-4.md:
```diff
@@ -22,9 +22,9 @@ This release:
+Please upgrade to `Armory extended Halyard 1.6.5`.
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-6-5.md:
```diff
@@ -19,7 +19,7 @@ This release addresses a problem where the Secrets config block was not generate
 - This release has no known issues
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-7-0.md:
```diff
@@ -20,7 +20,7 @@ This release has a bug which prevents `kubeconfig` files from being read from re
 Please upgrade to `halyard-armory` `1.7.1`
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-7-1.md:
```diff
@@ -17,7 +17,7 @@ This release fixes an issue which prevented `halyard` from reading local `kubeco
 No known issues
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-7-2.md:
```diff
@@ -48,7 +48,7 @@ hal deploy apply
 No known issues
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-8-0.md:
```diff
@@ -19,7 +19,7 @@ In addition to support for configuring `git/repo` artifact types, this release i
 No known issues
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-8-1.md:
```diff
@@ -14,7 +14,7 @@ aliases:
 No known issues
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-8-3.md:
```diff
@@ -8,7 +8,7 @@ aliases:
 ## 04/02/2020 Release Notes
 
-Armory Halyard 1.8.3 is the minimum version required to deploy Armory Spinnaker 2.19+. 
+Armory extended Halyard  1.8.3 is the minimum version required to deploy Armory 2.19+.
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-8-2.md:
```diff
@@ -14,7 +14,7 @@ aliases:
 No known issues
 
-## Halyard Armory Enterprise Spinnaker
+## Armory extended Halyard 
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-8-3.md:
```diff
@@ -34,10 +34,10 @@ Remove the top level key for Plugins in your Halconfig.
 
 **Workaround**
-Upgrade to Armory Halyard 1.9.0 or later.
+Upgrade to Armory extended Halyard 1.9.0 or later.
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-9-0.md:
```diff
@@ -8,25 +8,24 @@ aliases:
 ### Halyard version
 
-There is a known issue where Armory Halyard 1.9.0 fails to install Armory Spinnaker 2.18.1. The Pod for Echo enters a crash loop.
+There is a known issue where Armory extended Halyard 1.9.0 fails to install Armory 2.18.1. The Pod for Echo enters a crash loop.
```
Armory-extended Halyard

317brian replied:
> I got tired of highlighting "Armory extended Halyard"
> 
> RE the Operator. I think we should standardize on Armory Operator unless we are specifically talking about the open source version, which maybe we should call "Operator for Spinnaker" ?

We can probably safe call the oss version `Spinnaker Operator`. The LF would likely own any trademarks/copyrights for "Operator" in this context (if they exist). As a legal entity, the Spinnaker project exists under the LF umbrella, so it's using its own trademark.  There's also an established pattern of Kubernetes operators being called XYZ Operator.

aimeeu commented on content/en/docs/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-15-2.md:
```diff
@@ -11,7 +11,7 @@ aliases:
 
 ## Known Issues
-> Note: Updating to OSS 1.15.x created an issue with the secrets config in Clouddriver and Echo. Armory Halyard 1.6.5 fixes this issue. Please update your Halyard to 1.6.5 or later:
+> Note: Updating to OSS 1.15.x created an issue with the secrets config in Clouddriver and Echo. Armory extended Halyard 1.6.5 fixes this issue. Please update your Halyard to 1.6.5 or later:
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-15-1.md:
```diff
@@ -15,7 +15,7 @@ aliases:
 * `Clouddriver` caching of `Docker Registry` tags is slow in this release. Registries containing a large number of tags **(> 1000)** will result in `docker triggers` not properly triggering until caching is complete -- which might take 10 minutes or more.
 
-> Note: Updating to OSS 1.15.x created an issue with the secrets config in Clouddriver and Echo. Armory Halyard 1.6.5 fixes this issue. Please update your Halyard to 1.6.5 or later:
+> Note: Updating to OSS 1.15.x created an issue with the secrets config in Clouddriver and Echo. Armory extended Halyard 1.6.5 fixes this issue. Please update your Halyard to 1.6.5 or later:
```
Armory-extended Halyard

aimeeu commented on content/en/docs/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-19-5.md:
```diff
@@ -7,23 +7,23 @@ aliases:
 ### HTTP sessions for Gate
 
-Armory Spinnaker 2.19.x and later include an upgrade to the Spring Boot dependency. This requires you to flush all the Gate sessions for your Spinnaker deployment if you are upgrading from a version before 2.19.4. For more information, see [Flushing Gate Sessions](https://kb.armory.io/admin/flush-gate-sessions/).
+Armory 2.19.x and later include an upgrade to the Spring Boot dependency. This requires you to flush all the Gate sessions for your Spinnaker deployment if you are upgrading from a version before 2.19.4. For more information, see [Flushing Gate Sessions](https://kb.armory.io/admin/flush-gate-sessions/).
```
Should we use Armory rather than Spinnaker??

317brian commented:
@stuposluns the `Armory extended Halyard` lines exist because we were also using `Armory Halyard` like we were using `Armory Spinnaker`

aimeeu commented on content/en/docs/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-19-6.md:
```diff
@@ -52,13 +52,13 @@ Addressed a number of CVEs found within the Spinnaker services.
 **Plugins**
 
-This release supports Plugin deployment using Armory Halyard or the [Spinnaker Operator]({{< ref "operator" >}}). Consult the open source [Plugin](https://spinnaker.io/guides/user/plugins/) docs for Halyard usage or the [Plugins Operator Reference]({{< ref "plugins" >}}) for a manifest example.
+This release supports Plugin deployment using Armory extended Halyard or the [Spinnaker Operator]({{< ref "operator" >}}). Consult the open source [Plugin](https://spinnaker.io/guides/user/plugins/) docs for Halyard usage or the [Plugins Operator Reference]({{< ref "plugins" >}}) for a manifest example.
```
Should we change Spinnaker Operator to Armory Operator?

aimeeu reviewed (changes requested):
I got tired of highlighting "Armory extended Halyard"

RE the Operator. I think we should standardize on Armory Operator unless we are specifically talking about the open source version, which maybe we should call "Operator for Spinnaker" ?

317brian committed:
fix halyard name

317brian committed:
copyedits

aimeeu committed:
Replace Spinnaker in Installation dir; change 2 Armory Operator

Also update Operator to Armory Opertor unless specifically
talking about the open source Spinnaker Operator

aimeeu committed:
Merge branch 'master' into rename/bye-armory-spin

317brian committed:
Merge branch 'master' into rename/bye-armory-spin

317brian committed:
Merge branch 'master' into rename/bye-armory-spin

317brian commented:
Merging this since it gets us 90% of the way there.
